src/main/java/com/iridium/iridiumenchants/conditions/EntityType.java

```diff
@@ -0,0 +1,32 @@
+package com.iridium.iridiumenchants.conditions;
+
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.inventory.ItemStack;
+
+public class EntityType implements Condition {
+    @Override
+    public boolean apply(LivingEntity player, LivingEntity target, String[] args, ItemStack item) {
+        String operator = args[1];
+        String[] entities = args[2].split(",");
+
+        for (String entityType : entities) {
+            if (checkEntityType(target, entityType, operator)) {
+                return true;
+            }
+        }
```

Comment:
I dont believe this will work, lets say the Target is a Creeper, and we have `ENTITYTYPE != ZOMBIE, CREEPER`

It will loop over `ZOMBIE, CREEPER` it will say Zombie != Creeper = true, and it will return true.

So for the == condition, we want any of them to match (what we have already) but for != we want all of them to match